DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/CourseDelegates/Index.cshtml

```diff
@@ -33,15 +33,15 @@
              asp-route-sortDirection="@Model.CourseDetails?.SortDirection"
              asp-route-existingFilterString="@Model.CourseDetails?.ExistingFilterString"
              role="button">
-            Export current
+            Export
           </a>
         </div>
       </div>
 
       <div class="nhsuk-grid-row">
         <div class="nhsuk-grid-column-full">
           <p>
-            Choose a course to see delegates enrolled on it. If you'd like to export delegates on all courses, click "Export all" above.
+            Choose a course to see delegates enrolled on it. If you'd like to export delegates on all courses, click "Export" above.

```

Comment:
I think we can remove this entire second sentence, since it refers to the button that's no longer on this page?